tests/storage/export.go

```diff
@@ -1391,7 +1391,7 @@ var _ = SIGDescribe("Export", func() {
 	})
 
 	It("should mark the status phase skipped on VMSnapshot without volumes", func() {
-		vm := tests.NewRandomVMWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskCirros))
+		vm := libvmi.NewVirtualMachine(libvmi.NewCirros())
 		vm = createVM(vm)
```

Comment:
You will have to change `createVM` function to add the namespace.

from: `vm, err := virtClient.VirtualMachine(vm.Namespace).Create(context.Background(), vm)`
to: `vm, err := virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).Create(context.Background(), vm)`

Reply:
done. thanks.